client/tests/webdriver/baseSpecs/printReport.spec.js

```diff
@@ -124,4 +126,46 @@ describe("Show print report page", () => {
       }
     })
   })
+  describe("When on the print page of a report without classification", () => {
+    it("We should see the default text for classification in the header and footer banners", async() => {
+      await (await ShowReport.getClassificationHeader()).waitForExist()
+      await (await ShowReport.getClassificationHeader()).waitForDisplayed()
+      await (await ShowReport.getClassificationFooter()).waitForExist()
+      await (await ShowReport.getClassificationFooter()).waitForDisplayed()
+      expect(
+        await (await ShowReport.getClassificationHeader()).getText()
+      ).to.equal(DEFAULT_REPORT_CLASSIFICATION)
+      expect(
+        await (await ShowReport.getClassificationFooter()).getText()
+      ).to.equal(DEFAULT_REPORT_CLASSIFICATION)
+    })
```

Comment:
Nice work. One small remark: I would find it slightly more logical if these lines were reordered like this (also in the test below):
```javascript
      await (await ShowReport.getClassificationHeader()).waitForExist()
      await (await ShowReport.getClassificationHeader()).waitForDisplayed()
      expect(
        await (await ShowReport.getClassificationHeader()).getText()
      ).to.equal(DEFAULT_REPORT_CLASSIFICATION)
      await (await ShowReport.getClassificationFooter()).waitForExist()
      await (await ShowReport.getClassificationFooter()).waitForDisplayed()
      expect(
        await (await ShowReport.getClassificationFooter()).getText()
      ).to.equal(DEFAULT_REPORT_CLASSIFICATION)
    })
```